core/src/main/kotlin/com/ritense/valtimo/JobService.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ *  Licensed under EUPL, Version 1.2 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" basis,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.ritense.valtimo
+
+import java.time.ZonedDateTime
+import java.util.Date
+import mu.KotlinLogging
+import org.camunda.bpm.engine.ProcessEngine
+import org.camunda.bpm.engine.ProcessEngineException
+import org.camunda.bpm.engine.delegate.DelegateExecution
+import org.camunda.bpm.engine.runtime.Job
+
+class JobService(val processEngine: ProcessEngine) {
```

Comment:
It probably possible to inject the managementService directly in the constructor